### PR TITLE
tech debt

### PR DESCRIPTION
**Pull request by dlobrien**
added #1 swear jar phrase

### CODE REVIEW

**Review by Snepsts — CHANGES_REQUESTED** [review]
This is certainly a yellow card word, please refactor

File: index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const app = express();
 const client = new discord.Client();
 const port = 8080;
 const redCardList = ['shit', 'fuck', 'bitch', 'bitching',
-  'bitches', 'fucks', 'fucking', 'fleshlight']
+  'bitches', 'fucks', 'fucking', 'fleshlight', 'tech debt']
```
Comment (Seanage) [inline index.js:8]: it splits based on space. so this will never occur.

Comment (dlobrien) [reply index.js:8]: noooooooooo

Comment (Snepsts) [reply index.js:8]: Needs a major refactor to detect multiple words in a row

**Review by Seanage — CHANGES_REQUESTED** [review]
redCardList is intended for a list of words, not phrases. may be worthwhile to convert regex's against the entire body so that phrases can be used! I'll try this change

**Pull request closed without merging**